lib/index.js

```diff
@@ -34,6 +35,7 @@ bedrock.events.on('bedrock-session-http.session.get', (req, session) => {
 });
 
 bedrock.events.on('bedrock-express.configure.routes', app => {
+  app.use(boolParser());
```

Comment:
This will update the whole express app -- so not the right way to do it.

@davidlehn -- can you look at this and suggest the proper way?

Reply:
It looks to be just regular middleware.  This could just be added to endpoints that need it.  Unclear what the ramifications of adding it everywhere would be.